packages/constellation/src/components/Compositions/Table/SimpleTableExample/SimpleTableExample.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+import React, { FC, useState } from 'react'
+
+import { ButtonCell, Row, TableContainer, TextCell } from '../index'
+
+export default {
+  argTypes: { onClick: { action: 'clicked' } },
+  args: {},
+  title: 'Compositions/Table/SimpleTableExample',
+} as ComponentMeta<FC>
+
+const Template: ComponentStory<FC<{ onClick: () => void }>> = ({ onClick }) => {
+  const [sort, setSort] = useState<'up' | 'down'>('down')
+
+  return (
+    <TableContainer className='table-auto'>
+      <Row header>
+        <TextCell
+          text='Asset'
+          sort={sort}
+          toggleSort={() => setSort(sort === 'up' ? 'down' : 'up')}
```

Comment:
How will sorting work for tables? Is this something that will be coming later?

Reply:
Currently these components are expected to be used with something like `react-table` app side if sorting is required.

Reply:
Ok I see, is there any way you could describe this in the appropriate `types` and/or component file?